features/support/formatter_output_helpers.ts

```diff
@@ -0,0 +1,78 @@
+import _ from 'lodash'
+import { doesHaveValue, valueOrDefault } from '../../src/value_checker'
+import {
+  IJsonFeature,
+  IJsonScenario,
+  IJsonStep,
+} from '../../src/formatter/json_formatter'
+
+// Converting windows stack trace to posix and removing cwd
+//    C:\\project\\path\\features\\support/code.js
+//      becomes
+//    features/support/code.js
+function normalizeExceptionAndUri(exception: string, cwd: string): string {
+  return exception
+    .replace(cwd, '')
+    .replace(/\\/g, '/')
+    .replace('/features', 'features')
+}
+
+function normalizeProtobufObject(obj: any, cwd: string): void {
```

Comment:
should obj be an object rather than any?

Reply:
Using object produces errors as we are referencing directly. Going to stick with any for now